packages/client/src/components/ActivityTable.spec.js

```diff
@@ -0,0 +1,39 @@
+import {
+  beforeEach, describe, it, expect,
+} from 'vitest';
+import Vuex from 'vuex';
+import { createLocalVue, shallowMount } from '@vue/test-utils';
+import ActivityTable from '@/components/ActivityTable.vue';
+
+const localVue = createLocalVue();
+localVue.use(Vuex);
+let wrapper;
+const stubs = ['b-table', 'b-icon', 'b-iconstack'];
```

Comment:
bit confused about the pattern of stubbing these out

Reply:
omitting stubs entirely doesn't seem to change the behavior anyways tho, at least with current setup

Reply:
I agree, you can omit these stubs. (They might provide some minor performance benefit in preventing a full mount from rendering everything, but it's pretty negligible overall.) And, in fact, for your test you need `b-table` to render to evaluate what you want to test.

Reply:
thanks for confirming - yeah, what you're seeing was my latest iteration of throwing spaghetti at the wall and seeing if the behavior of the test changed to try to understand it better. Most of the time, I swear I wasn't stubbing out the `b-table` :D